Example/Backpack/View Controllers/ChipsViewController.swift

```diff
@@ -122,6 +123,9 @@ extension ChipsViewController: UICollectionViewDelegate {
         if backgroundTint != nil {
             cell.backgroundTint = backgroundTint
         }
+        if style == style {
```

Comment:
is this checking `style != nil`?

Reply:
LOL I meant `if let style = style` 🙈 
good catch!